RecoBTag/SecondaryVertex/interface/CombinedSVComputer.h

```diff
@@ -221,10 +221,13 @@ void CombinedSVComputer::fillCommonVariables(reco::TaggingVariableList & vars, r
                 vars.insert(btau::trackMomentum, trackMag, true);
                 vars.insert(btau::trackEta, trackMom.Eta(), true);
 
-                vars.insert(btau::trackPtRel, VectorUtil::Perp(trackMom, jetDir), true);
+                // check for erroneous Perp^2 values before evaluating Perp (fix for DeepCSV NaN outputs)
+                double Perp_trackMom_jetDir = VectorUtil::Perp2(trackMom, jetDir) > 0. ? VectorUtil::Perp(trackMom, jetDir) : 0.;
```

Comment:
Use lower case initials for the local variables:  ```perp_trackMom_jetDir```

A (tiny) optimization could be (compute the scalar product only once):
```
double perp_trackMom_jetDir = VectorUtil::Perp2(trackMom, jetDir);
perp_trackMom_jetDir = (perp_trackMom_jetDir > 0. ? std::sqrt(perp_trackMom_jetDir ) : 0. ); 
```

Reply:
@perrotta Thanks! Updated.